structured_experiments/2022.04.28.notebook_template/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM scribble-ocr-train as tesseract_base_image
+
+ADD requirements.txt /tmp/requirements.txt
+RUN pip install -r /tmp/requirements.txt
+COPY mytesseract mytesseract

```

Comment:
Would be great to have a comment on how this line allows to copy files into the container

Reply:
I would check the  [Wiki](https://github.com/BloomTech-Labs/scribble-stadium-ds/wiki/Contribution-Guide#components-of-a-structured-experiment), I want to say this info is in there 👍